documentation/modules/con-securing-kafka-bridge.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-kafka-bridge-overview.adoc
+
+[id='con-securing-kafka-bridge-{context}']
+= Securing the Kafka Bridge
+
+[role="_abstract"]
+Authentication and encryption between HTTP clients and the Kafka Bridge is not supported.
+Requests sent from clients to the Kafka Bridge are sent without authentication or encryption.
+Requests must use HTTP rather than HTTPS.
+
+You can configure the following between the Kafka Bridge and your Kafka cluster:
+
+* TLS or SASL-based authentication
+* A TLS-encrypted connection
+
+You configure the Kafka Bridge for authentication through its xref:proc-configuring-kafka-bridge-{context}[properties file].
+
+You can also use ACLs in Kafka brokers to restrict the topics that can be consumed and produced using the Kafka Bridge.
+
+== Securing the Kafka Bridge on Kubernetes
+
+If you have deployed the Kafka Bridge on a Kubernetes cluster, you can use the following methods to secure a Kafka Bridge deployment on a Kubernetes cluster.
+
+* Kubernetes network policies that define which pods can access the Kafka Bridge
+* Reverse proxies, such as OAuth 2.0
+* API gateways
+* Ingress or OpenShift routes with TLS termination
```

Comment:
I think this should be made a bit more generic and leave out the Kubernetes part. I think it should also not be a subchapter of 1.3 => these are a bit different aspects => Securing the KAfka communication & securing the HTTP part. Something like this perhaps?

```suggestion
[id='con-securing-kafka-bridge-{context}']
= Securing connectivity to the Kafka cluster

You can configure the following between the Kafka Bridge and your Kafka cluster:

* TLS or SASL-based authentication
* A TLS-encrypted connection

You configure the Kafka Bridge for authentication through its xref:proc-configuring-kafka-bridge-{context}[properties file].

You can also use ACLs in Kafka brokers to restrict the topics that can be consumed and produced using the Kafka Bridge.

= Securing the Kafka Bridge HTTP interface

[role="_abstract"]
Authentication and encryption between HTTP clients and the Kafka Bridge is not supported directly in the Kafka Bridge.
Requests sent from clients to the Kafka Bridge are sent without authentication or encryption.
Requests must use HTTP rather than HTTPS.

You can combine the Kafka Bridge with other tools to secure it:

* Network policies and firewalls which define who can connect to it
* Reverse proxies, such as OAuth 2.0
* API gateways
```

Reply:
Okay, thanks. I've made the split.